nav.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/0a707a45d9.js" crossorigin="anonymous"></script>
+    <title>Navigation</title>
+</head>
+<body>
+    <section class="nav-header">
+        <span class="nav-icon"><i class="fas fa-book-reader"></i></span> 
+        <div class="nav-main-div"><a class="btn btn-nav-home" href="#">Home </a>
+        </div>
+       <div class="nav-extra-div">
+        <a class="btn btn-nav-pricing" href="#">Pricing </a>
+        <a class="btn btn-nav-faq" href="#">FAQs </a>
+        <a class="btn btn-nav-about" href="#"> About</a>
+      </div> 
+        <img src="assets/avatar.svg" alt="Avatar" class="avatar avatar-sm"/>
+       
+    </section>
```

Comment:
Use `nav` instead of `section`. there could be better names than `nav-extra-div` such as `nav-items-container`.